.taskcluster.yml

```diff
@@ -40,151 +37,170 @@ tasks:
       $if: 'tasks_for == "github-pull-request"'
       then: ${event.pull_request.head.repo.html_url}
       else: ${event.repository.html_url}
+
+    is_repo_trusted:
```

Comment:
@mitchhentges I remember you were having some difficulties with booleans in a previous patch, is this going to work?

Reply:
Yeah, Mihai is being smarter than me and making `is_repo_trusted` use an `$if/else`, rather than just placing the string inside like a goofus.
Looks good :+1: